server/jest/fixtures.js

```diff
@@ -45,6 +45,31 @@ const authorizationRequestCancelledEventBody = {
   authorizationLevel: 'READ'
 }
 
+const authorizationInviteCancelledEventBody = {
+  actorId: "user8888",
+  actorName: "testing!",
+  granteeType: "USER",
+  granteeEmail: "patrick@star.com",
+  granteeId: "patrick",
+  granteeName: "Patrick Star",
+  resourceId: "use-case-discvoerable",
+  resourceName: "use-case-discvoerable",
+  resourceOwner: "use-case-org",
+  resourceOwnerName: "Use Case Org",
+  resourceUrl: "http://localhost:3000/use-case-org/use-case-discvoerable",
+  event: {
+    triggeredBy: "user8888",
+    created: "Sat, 12 Dec 2020 21:36:48 GMT"
+  },
+  eventType: "dataset.authorization_invite.created",
```

Comment:
Am I misunderstanding what this fixture is for? Should the event type be "created" for a cancelled event?

Reply:
Good catch, the event type should be "cancelled" here.

Reply:
Looking at it again, I think the content of the fixture was right, I just misnamed it. I wanted to test that the `cancel` button was showing correctly on authorization invites.

Reply:
That makes more sense. The event type here should be driving the functionality.